lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -12,6 +12,8 @@
 from matplotlib import cm, rcParams
 from matplotlib.backends.backend_pdf import PdfPages
 from matplotlib import pyplot as plt
+from matplotlib.testing.determinism import (_test_source_date_epoch,
```

Comment:
There is common code in PDF and PS backends tests, that I wrote in `determinism.py`. Where do you think it should be?
I renamed the functions to remove the `test' keyword